### PR TITLE
7682 store acccount ids and applications in cr

### PR DESCRIPTION
**Pull request by markmckinnon**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/centralrepository/application/OtherOccurrences.java [hunk 1]
```diff
@@ -69,6 +73,43 @@
     private OtherOccurrences() {
     }
 
+    public static Collection<CorrelationAttributeInstance> getCorrelationAttributeFromOsAccount(Node node, OsAccount osAccount) {
```
Comment (wschaeferB) [inline Core/src/org/sleuthkit/autopsy/centralrepository/application/OtherOccurrences.java:76]: comment for method

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoDbUpgrader15To16.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
+/*
+ * Central Repository
+ *
+ * Copyright 2020 Basis Technology Corp.
```
Comment (wschaeferB) [inline Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoDbUpgrader15To16.java:4]: date

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoDbUpgrader15To16.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+ */
+package org.sleuthkit.autopsy.centralrepository.datamodel;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import org.sleuthkit.datamodel.CaseDbSchemaVersionNumber;
+
+/**
+ * This class updates CR schema to 1.5
```
Comment (wschaeferB) [inline Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoDbUpgrader15To16.java:27]: copy paste version number error

File: Core/src/org/sleuthkit/autopsy/centralrepository/eventlisteners/CaseEventListener.java [hunk 4]
```diff
@@ -646,6 +659,94 @@ public void run() {
         } // CURRENT_CASE
     }
 
+    @NbBundle.Messages({"CaseEventsListener.module.name=Central Repository",
+        "CaseEventsListener.prevCaseComment.text=Users seen in previous cases",
+        "CaseEventsListener.prevExists.text=Previously Seen Users (Central Repository)"})
+    private final class OsAccountInstancesAddedTask implements Runnable {
```
Comment (wschaeferB) [inline Core/src/org/sleuthkit/autopsy/centralrepository/eventlisteners/CaseEventListener.java:665]: Class comment

**Review by wschaeferB — CHANGES_REQUESTED** [review]

**Comment by wschaeferB** [comment]
Everything works, just some comment changes to clean up.

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoDbUpgrader15To16.java (hunk 3 above)
Comment (markmckinnon) [reply Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoDbUpgrader15To16.java]: Updated to 1.6

File: Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoDbUpgrader15To16.java (hunk 2 above)
Comment (markmckinnon) [reply Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoDbUpgrader15To16.java]: Changed to 2021

File: Core/src/org/sleuthkit/autopsy/centralrepository/application/OtherOccurrences.java (hunk 1 above)
Comment (markmckinnon) [reply Core/src/org/sleuthkit/autopsy/centralrepository/application/OtherOccurrences.java:84]: Added comment

**Review by wschaeferB — APPROVED** [review]

**Pull request merged**